pegasus/helpers/section_api_helpers.rb

```diff
@@ -555,7 +555,8 @@ def to_owner_hash
       script: script,
       course_id: @row[:course_id],
       teachers: teachers,
-      students: students
+      students: students,
+      studentNames: students.map(&:name),
```

Comment:
Are we still using both API's? Do we depend on them returning the same things? (i.e. why does pegasus need to return studentNames now).